src/Voron/Impl/Compaction/StorageCompaction.cs

```diff
@@ -486,7 +486,7 @@ private static long CopyTableTree(StorageEnvironment compactedEnv,
                 {
                     long transactionSize = 0L;
 
-                    Table inputTable = innerTxr.OpenTable(schema, treeName);
+                    Table inputTable = innerTxr.OpenTable(schema, treeName, prefetch: true);
```

Comment:
As I see it's the only place where we enable prefetching for a table. Does it mean that the prefetching is disabled effectively after this change?

Reply:
No it is not disabled overall. What it is disabled is speculative prefetching, which is essentially the type of prefetching that happens at iterators and the one that caused the time series issue. The speculative prefetching is overly aggressive so it should be handled with care, with newer and faster disks it is not so necessary anymore, except in cases like compaction which the time we spend offline is a liability. There may be a few other cases out there which would make sense, but the correct approach is to treat them in a case-by-case basis.

The decision to effectively shutdown speculative prefetching for normal operation is because it causes IO spikes, and we don't like IO spikes. The normal per page advanced prefetching is still in place.

Reply:
What would be your personal recommendation for moving that to 6.2?

Reply:
Not a big deal, it would probably rebase correctly. However, being LTS this is the kind of change that probably is not a good idea to push (too low level).

Reply:
@grisha-kotler regarding your question:

> Can this go to 6.2?

I rely on above @redknightlois'  suggestion and agree here.